### PR TITLE
Add Culdcept Revolt

### PR DESCRIPTION
**Pull request by lorrampi**
Adds Culdcept Revolt.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/culdcept-revolt/game.dat [hunk 1]
```diff
@@ -0,0 +1,19 @@
+title = "Culdcept Revolt"
+description = "As the Culdcept series celebrates its 20th Anniversary, the addictive card-meets-board game is back, bigger and better than ever with hundreds of new cards, inventive boards, and exciting multiplayer gameplay! Follow Allen, a new recruit for the rebel Free Bat forces as you fight for freedom from the tyranny of Count Kraniss. Build your deck, create new strategies, and dominate the board in Culdcept Revolt!"
```
Comment (Leo626) [inline games/culdcept-revolt/game.dat]: Use the description from wikipedia like this:

> Culdcept Revolt is a turn-based strategy video game developed by OmiyaSoft and published by NIS America in North America and Europe, and Nintendo in Japan. It is the sequel of Culdcept, though the two games have different plotlines.

**Comment by Leo626** [comment]
@lorrampi Please address the description. It's the last thing preventing this from getting merged.

File: games/culdcept-revolt/game.dat (hunk 1 above)
Comment (lorrampi) [reply games/culdcept-revolt/game.dat]: The description from wikipedia, really it got better.

**Comment by lorrampi** [comment]
@Leo626 how do I do this? I'm a beginner here, but I want to help with whatever it takes...

**Comment by Leo626** [comment]
You can make the changes in your repository directly in Github [here](https://github.com/lorrampi/citra-games-wiki/blob/pr-builder-culdcept-revolt-69/games/culdcept-revolt/game.dat).

**Comment by lorrampi** [comment]
@Leo626 it worked? Is there anything else I need to do?

**Pull request merged**